_episodes/07-Practical Considerations:Reporting-results.md

```diff
@@ -59,7 +59,7 @@ These 4 components can be used to calculate the most common metrics in supervise
 
 In some use-cases, such as *information retrieval tasks*, the **precision** of a model can be important where you aren't expecting to evaluate any negative cases. For example if we wrote an ML algorithm to scan clinic letters for people with epilepsy, we know that most of the population do not have epilepsy and so "no epilepsy" won't be explicitly recorded. Only *if* someone has epilepsy would it be recorded. So we are interested in assessing out of all the cases the algorithm predicts as having epilepsy, how many actually had epilepsy? but we are also interested in assessing out of all the *possible* epilepsies in the population, what percentage does the algorithm pick up? The latter is measured by precision. Precision is used for *information retrieval* tasks, and sensitivity is used for *classification* tasks. 
 
-While it might seem like accuracy, a combination of sensitivity and specificity, is the most important metric, it actually is the least useful. If your primary aim is to pick up as many COVID-19 cases as possible to reduce spread, *sensitivity* will tell you how well your algorithm does, not accuracy. If it is important to be as sure as possible that someone will likely develop disease because it will likely cause harm in other ways i.e. developing mental health issues due a diagnosis of dementia, *specificity* can help you prioritize when to label someone with the disease. Hence the main task in supervised ML is knowing how to trade sensitivity with specificity. This is where the **Receiver Operator Characteristic (ROC) Curve** can help.
+Performance metrics are problem-dependent. For example, if your primary aim is to pick up as many COVID-19 cases as possible to reduce spread, *sensitivity* will tell you how well your algorithm does, not accuracy. If it is important to be as sure as possible that someone will likely develop disease because it will likely cause harm in other ways i.e. developing mental health issues due a diagnosis of dementia, *specificity* can help you prioritize when to label someone with the disease. Therefore, it is important to know how to trade sensitivity with specificity. This is where the **Receiver Operator Characteristic (ROC) Curve** can help.
```

Comment:
@arronlacey I don't fully understand this sentence:

```
If it is important to be as sure as possible that someone will likely develop disease because it will likely cause harm in other ways
```

Reply:
yeah that sentence is way too convoluted. I was trying to get at it being bad to falsely labeleveryone with dementia and create a potential mental health crisis because everyone is worried. I'll think have a think! Maybe something to do with avoiding unecessary, expensive and invasvie biopsies based on a false positive test.

Reply:
👍